angular-ui-router/angular-ui-router.d.ts

```diff
@@ -168,6 +168,17 @@ declare module angular.ui {
         current: IState;
         params: IStateParamsService;
         reload(): void;
+        
+        $current: IStateServiceUtilities;
+    }
+    
+    interface IStateServiceUtilities {
```

Comment:
I suggested to rename stateserviceutilities to resolvedstate. This is because it felt to me that $current is in fact a resolved instance of an state declaration.
I have no evidence of this, just a hunch, based on the fact that name field is the same.